app/frontend/src/assets/scss/style.scss

```diff
@@ -264,15 +264,23 @@ a,
   }
 
   // form inputs
-  .form-control {
+  .form-control:not(.is-invalid) {
+    // input borders
     border: 1px solid $bcgov-input-placeholder;
+
+    // placeholder text
     &::placeholder,
     .choices__placeholder {
       color: $bcgov-input-placeholder;
       opacity: inherit;
     }
   }
 
+  .form-control.is-invalid,
+  .was-validated .form-control:invalid {
+    background-image: none;
+  }
+
```

Comment:
I think we need to be really careful with this. Are there other background images used for other formio functionality that having a `background-image: none` could override? Like some other control using an icon that's neccessary and having it disappear on validation could be unintended behaviour.

What's the intent of removing this as well? It's a FormIO default isn't it. We've decided not to mess around with basic FormIO behaviour of the rendered controls.

Personally I'd rather stay away from removing things like this unless it's an absolute requirement.
@parc-jason you've played more with the actual formIO controls than I have.

Reply:
yes. good point. maybe we can just apply this to select fields.
this was done to address the issue with the red circle icon appearing over the dropdown arrow. 
![image](https://user-images.githubusercontent.com/58443522/96942026-ce0d0680-1488-11eb-9d63-cd0e66314984.png)

Reply:
@loneil  - i don't really know what styles etc apply to formio.  but I agree, we have to be super particular with any style override.  between vuetify and bootstrap and formio, there are a lot of implications.  i think we are going to have to accept that not everything will look like the bc gov design system, and any style/class override that must be done (like above) has to be done in the most specific way possible.  the finest grained scope that will correct the issue.